app/routes/wallet/credentials.py

```diff
@@ -136,38 +246,96 @@ async def get_credential_revocation_status(
     return result
 
 
-@router.get("/w3c", response_model=VCRecordList)
+@router.get(
+    "/list/w3c",
+    response_model=VCRecordList,
+    summary="Fetch a list of W3C credentials from the wallet",
+)
 async def list_w3c_credentials(
-    count: Optional[str] = None,
-    start: Optional[str] = None,
-    wql: Optional[str] = None,
-    body: Optional[W3CCredentialsListRequest] = None,
+    contexts: Optional[List[str]] = Query(None),
+    types: Optional[List[str]] = Query(None),
+    schema_ids: Optional[List[str]] = Query(None),
+    subject_ids: Optional[List[str]] = Query(None),
+    proof_types: Optional[List[str]] = Query(None),
+    issuer_id: Optional[str] = Query(None),
+    given_id: Optional[str] = Query(None),
+    tag_query: Optional[str] = Query(None),
+    max_results: Optional[int] = Query(None),
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> VCRecordList:
-    """Fetch a list of W3C credentials from the wallet."""
+    """
+    Fetch a list of W3C credentials from the wallet
+    ---
+    Credential_id and record_id are duplicates of each other.
+    The W3C credentials can be filtered by the parameters provided.
+
+    Parameters:
+    ---
+        contexts: Optional[List[str]]
+        types: Optional[List[str]]
+        schema_ids: Optional[List[str]]
+        issuer_id: Optional[str]
+        subject_ids: Optional[List[str]]
+        given_id: Optional[str]
+        proof_types: Optional[List[str]]
+        tag_query: Optional[str]
+        max_results: Optional[int]
```

Comment:
In hierdie  gevalle waar alles optional is, verkies ek om net "Parameters" rename na "Optional Parameters", and die Optional[..] te drop